raiden_contracts/contracts/test/StandardToken.sol

```diff
@@ -11,7 +11,7 @@ import "raiden/Token.sol";
 
 contract StandardToken is Token {
     uint256 internal _total_supply;
-    mapping (address => uint256) balances;
+    mapping (address => uint256) public balances;
```

Comment:
This is actually not needed, because we have `balanceOf` and this is the correct interface to use. Adding another public interface is confusing.
Anyway, the token contracts are messed up, so you can leave it like this - opened https://github.com/raiden-network/raiden-contracts/issues/51 for them.

Reply:
ah, you are right, this should not be here. I was just playing with the contract source and forgot to revert to the original version.